paimon-arrow/src/test/java/org/apache/paimon/arrow/vector/ArrowFormatWriterTest.java

```diff
@@ -204,7 +204,7 @@ public void testCWriter() {
 
     @ParameterizedTest
     @ValueSource(booleans = {false, true})
-    public void testWriteWithExternalAllocator(boolean allocationFailed) {
+    void testWriteWithExternalAllocator(boolean allocationFailed) {
```

Comment:
I noticed that there are more test cases in this class. Why is only this test case's visibility is changed?

Reply:
I may have missed them, I'm checking them one by one.